src/DynamoCore/BuiltInAndOperators/Operators.xml

```diff
@@ -110,9 +110,10 @@
         </member>
 		<member name="%Not">
 		            <summary>
-                !x
+                Reverses the result, returns false if the result is true
             </summary>
-            <param name="x">x value.</param>
+            <param name="x">boolean to reverse.</param>
+            <returns name="boolean">reversed boolean</returns>
```

Comment:
type `var[]..[]` in description

Reply:
Addressed in 2b6a2d7
output description: `type var[]..[] (reversed boolean)`